blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/Configuration.java

```diff
@@ -81,36 +95,50 @@ public int getIntegerProperty(Enum<? extends ConfigDefaults> name) {
         return getIntegerProperty(name.toString());
     }
```

Comment:
Do not see much value in the following new methods that were added. intFromString, longFromString, floatFromString.